### DISCUSSION TITLE
OdbError in TestRepository while running tests locally

### DISCUSSION THREAD

**Discussion by ajhoward2** [github-linguist/linguist#6485] [Q&A] [answered]
When running the tests locally I receive a few errors, all similar to following:

Running command: `sudo bundle exec rake test`

Partial Output:
```
 7) Error:
TestRepository#test_linguist_override_documentation?:
Rugged::OdbError: object not found - no match for id (01d6b9c637a7a6581fe456c600725b68f355b295)
    /home/ajhoward/linguist/lib/linguist/repository.rb:128:in `lookup'
    /home/ajhoward/linguist/lib/linguist/repository.rb:128:in `current_tree'
    /home/ajhoward/linguist/lib/linguist/repository.rb:123:in `read_index'
    /home/ajhoward/linguist/test/test_repository.rb:122:in `test_linguist_override_documentation?'

1709 runs, 34720 assertions, 0 failures, 7 errors, 1 skips
```

I originally did a clone of the repo directly and had no failures or errors but when I went back and forked the repo and cloned my fork, I then started receiving these errors. I can see that Linguist::Repo is created using the master OID, and the rugged repo which references the .git folder on my local repo. I am new to GitHub, but I was thinking that the .git would carry the commits from the original repo that I forked, so this should not be an issue when a Commit.lookup is performed.

I followed the CONTRIBUTING.md page for adding my language and am looking for some more information regarding this error and any pointers in the right direction to debug this issue. Please let me know if you need more information, or if I should go ahead and start a PR so this issue can be reviewed. Thanks!

**Comment by lildude** [comment, marked as answer]
This error happens when you've only forked the master branch (the `Copy the master branch only` checkbox was checked at the time, which is the default). This test needs the `test/attributes` branch so you need to either refork or perform a full pull from the origin repo and push to your fork.
